pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -120,7 +119,8 @@ func NewOperationExecutor(
 	kubeClient internalclientset.Interface,
 	volumePluginMgr *volume.VolumePluginMgr,
 	recorder record.EventRecorder,
-	checkNodeCapabilitiesBeforeMount bool) OperationExecutor {
+	checkNodeCapabilitiesBeforeMount bool,
```

Comment:
Will you add the new `operation_executor_test.go` tests and fake version of the generator in a separate PR? Since there are no other PRs depending on this behavior, I would prefer having both those in this PR as well (maybe as separate commits).